src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -2869,6 +2869,31 @@ public static unsafe void ConstructorsTest_InvalidUTF8()
             }
         }
 
+        [Fact]
+        public static unsafe void ConstructorsTest_InvalidArguments ()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("value", () => new String ((sbyte*) null, 0, 1, Encoding.Default);
+            AssertExtensions.Throws<ArgumentNullException>("value", () => new String ((sbyte*) null, 1, 0, Encoding.Default);
```

Comment:
Nit: The typical CoreFX coding style is to not have space between function name and arguments (ie `new String(`).